cl/search/documents.py

```diff
@@ -1247,8 +1251,10 @@ def prepare_parties(self, instance):
         if not out["party"]:
             # Get party from docket case_name if no normalized parties are
             # available.
-            party_from_case_name = get_parties_from_case_name(
-                instance.case_name
+            party_from_case_name = (
+                get_parties_from_bankruptcy_case_name(instance.case_name)
+                if instance.court_id.endswith("b")
```

Comment:
I really thought we had an `is_bankruptcy_court` helper somewhere, no? This feels a bit crude.

Reply:
@mlissner I also thought we had an is_bankruptcy_court helper.  I've checked the Court model and its custom manager, but I haven't found one.  I did discover a [query](https://github.com/freelawproject/courtlistener/blob/f808c2d86421393f597e47219cf0ba784064a9ef/cl/search/models.py#L1921-L1924) that retrieves all bankruptcy courts.

I think we can add a small helper function, similar to the existing `is_appellate_court` helper.

```python
def is_bankruptcy_court(court_id: str) -> bool:
    bankr_court_ids = Court.federal_courts.bankruptcy_courts()
    return bankr_court_ids.filter(pk=court_id).exists()
```

Let me know what you think.

Reply:
Looks good to me. Checking for `endswith('b')` probably works, until it doesn't, so it's worth a tiny DB query.